src/utils.js

```diff
@@ -73,6 +39,29 @@ export function getSides(sides?: string = ''): Array<IndividualSides> {
   return uniq(allSides)
 }
 
+export function getMarginClasses(
+  margin?: string,
+  marginSize?: MarginSizes
+): Array<string | void> {
+  const marginSizeClasses = {
```

Comment:
I think we should move this out so we aren't creating a new object essentially on every render